### PR TITLE
tests: Remove unused mininode functions {ser,deser}_int_vector(...). Remove unused imports.

### PR DESCRIPTION
**Pull request by practicalswift**
* Remove unused mininode functions `deser_int_vector(f)` and `ser_int_vector(l)`. Last use removed in 3858aabbd084b2cc4534c40a178bbcc22fb82968. Friendly ping @jnewbery :-)
* Remove unused imports.

### CODE REVIEW

**Comment by promag** [comment]
utACK 5122755.

**Comment by jnewbery** [comment]
Thanks @practicalswift . Looks good but conflicts with #11648. Do you mind reviewing that first to avoid a rebase?

**Comment by practicalswift** [comment]
@jnewbery Sure!

Should I cherry-pick f9cd9b1bfaf65b4e24c1eb9ca8268513e78cc458 into this PR, or should I await merge of #11648?

**Comment by jnewbery** [comment]
Thanks! I don't think you need to do anything for now. If #11648 can get merged soon, then I'd just wait for that and rebase this on top. If #11648 takes a while, then it's fine for this to be merged and I'll rebase #11648 on top of this.

**Comment by practicalswift** [comment]
@jnewbery Great! 👍

**Comment by practicalswift** [comment]
Rebased! :-)

**Comment by jnewbery** [comment]
Tested ACK f522fb7c9c121641ce552774e3993942106ec336

File: test/functional/minchainwork.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import time
 
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import sync_blocks, connect_nodes, assert_equal
+from test_framework.util import connect_nodes, assert_equal
```
Comment (promag) [inline test/functional/minchainwork.py]: Do we sort here?

**Review by promag — COMMENTED** [review]
Tested ACK.

**Comment by MarcoFalke** [comment]
utACK f522fb7

**Comment by fanquake** [comment]
utACK f522fb7

**Pull request merged**